common.sh

```diff
@@ -107,23 +89,28 @@ FFMPEG_CONFIGURE_FLAGS=(
   --enable-demuxer=xwma
   --enable-encoder=flac
   --enable-encoder=libopus
+  --enable-encoder=libmp3lame
   --enable-encoder=pcm*
   --enable-ffmpeg
   --enable-ffprobe
```

Comment:
This is unrelated to this PR, but can we remove --enable-ffprobe? Although it wouldn't reduce ffmpeg binary size, it could probably speed up CI